cirq/ops/pauli_string.py

```diff
@@ -794,6 +808,38 @@ def conjugated_by(self, clifford: 'cirq.OP_TREE') -> 'PauliString':
         coef = -self._coefficient if should_negate else self.coefficient
         return PauliString(qubit_pauli_map=pauli_map, coefficient=coef)
 
+    def after(self, ops: 'cirq.OP_TREE') -> 'cirq.PauliString':
+        r"""Determines the equivalent pauli string after some operations.
+
+        If the PauliString is $P$ and the Clifford operation is $C$, then the
+        result is $C P C^\dagger$.
+
+        Args:
+            ops: A stabilizer operation or nested collection of stabilizer
+                operations.
```

Comment:
Ah, this is actually.. sort of intentional? For example,  reset is a stabilizer operation. Is it allowed? Well, currently, it's not implemented. But in the future I intend to have `cirq.Z(q).before(cirq.reset(q))` return the identity pauli string. But *but* `cirq.X(q).before(cirq.reset(q))` is not defined, so that will remain disallowed. So there is an "is it implemented yet" time element present, and also state-dependent error modes. I'm describing what I want the method to be.

I don't consider the 2 state to be part of the stabilizer group that is being referred to (disambiguated by the paulis that are allowed).

Reply:
Fair enough. Vagueness does have its uses. SGTM.